gadget/update.go

```diff
@@ -43,7 +43,10 @@ type GadgetData struct {
 	// Info is the gadget metadata
 	Info *Info
 	// RootDir is the root directory of gadget snap data
+	// XXX: should be GadgetRootDir
 	RootDir string
+	// KernelRootDir is the root directory of kernel snap data
+	KernelRootDir string
```

Comment:
Please clarify which kernel this is, specifically in the situation when "snap refresh" runs and both the kernel and gadget are being refreshed.